src/sif/hierarchylimits.cc

```diff
@@ -0,0 +1,7 @@
+#include "sif/hierarchylimits.h"
+
+using namespace valhalla::sif;
+
+bool HierarchyLimits::StopExpanding(const float dist) const {
```

Comment:
so the motivation for this was because, when you want to disable this for testing recompiling is a pain in the butt, yes?

Reply:
Aye, the header file change invalidated _a lot_ of files.